tox.ini

```diff
@@ -70,7 +70,7 @@ max-line-length=88
 
 [testenv:black]
 deps = black==22.12.0
-commands = black --extend-exclude ^.*\b(migrations)\b.*$ --check .
```

Comment:
Why?

Reply:
Ups, good catch, I have a different version of black locally so running it to fix the issues makes different changes and I just remove the `--check` to fix them with `make format`